### PR TITLE
Unify already on IAM version messaging and only migrate if necessary in IAM v1.0

### PR DESCRIPTION
**Pull request by tylercloke**
### :nut_and_bolt: Description: What code changed, and why?

Updated the messaging output for trying to switch to an IAM version you are already on to be consistent. Also updated the reset to v1 GRPC command to return `AlreadyExists` similar to the upgrade to v2 command and not run the reset migration. 

### :athletic_shoe: How to Build and Test the Change

```
rebuild components/automate-cli
rebuild components/authz-service
```

Upgrade, reset, etc.

### :white_check_mark: Checklist

- [ ] Tests added/updated?
- [ ] Docs added/updated?

### :camera: Screenshots, if applicable

### CODE REVIEW

**Comment by susanev** [comment]
theres a card for this https://github.com/chef/automate/issues/1061

**Comment by tylercloke** [comment]
@susanev yep, doing it! check the branch name :) this is a draft PR so it's not all linked etc yet.

File: components/authz-service/server/v2/policy_test.go [hunk 1]
```diff
@@ -2606,21 +2606,23 @@ func TestResetToV1(t *testing.T) {
 	status := ts.status
 
 	cases := map[string]func(*testing.T){
-		"initial migration status: reset keeps pristine status": func(t *testing.T) {
+		"initial migration status eturns AlreadyExists to indicate no-op": func(t *testing.T) {
```
Comment (blakestier) [inline components/authz-service/server/v2/policy_test.go]: missing the `r` on `eturn`

File: components/automate-cli/cmd/chef-automate/iam.go [hunk 2]
```diff
@@ -150,12 +150,7 @@ func init() {
 }
 
 // Note: the indentation is to keep this in line with writer.Body()
-const alreadyMigratedMessage = `You have already upgraded to IAM %s.
-  If you wish to re-run the migration, first run:
-
-      chef-automate iam reset-to-v1
-
-  Then re-run this command.`
+const alreadyMigratedMessage = `You are already using IAM %s.`
```
Comment (blakestier) [inline components/automate-cli/cmd/chef-automate/iam.go]: nit: something sounds weird about "using" a version. What do you think about `"You are already on IAM version %s."`?

Comment (susanev) [reply components/automate-cli/cmd/chef-automate/iam.go]: I support this change!

Comment (susanev) [reply components/automate-cli/cmd/chef-automate/iam.go]: I support this change!

File: components/authz-service/server/v2/policy.go [hunk 3]
```diff
@@ -648,6 +648,7 @@ func (s *policyServer) ResetToV1(ctx context.Context,
 	}
 	switch ms {
 	case storage.Pristine: // skip
```
Comment (srenatus) [inline components/authz-service/server/v2/policy.go]: This comment isn't accurate anymore, now, is it? 😉

File: components/automate-cli/cmd/chef-automate/iam.go (hunk 2 above)
Comment (srenatus) [inline components/automate-cli/cmd/chef-automate/iam.go]: [nit] No need for backticks if we don't include a newline or `"`s

**Comment by srenatus** [comment]
fixed and amended some tests, will merge when green

**Pull request merged**